### DISCUSSION TITLE
Spring Cloud AWS v3.x and @Value not working

### DISCUSSION THREAD

**Discussion by Cimballi** [awspring/spring-cloud-aws#800] [Q&A]
I recently migrated a project from Spring Cloud AWS v2.x to v3.x and I am facing a problem with @Value.
In the logs I can see :
"Populating property retrieved from AWS Parameter Store: .x.y.z"
But then later :
Could not resolve placeholder 'x.y.z' in value \"${x.y.z}

Other properties, like Spring properties for the database configuration for example, are working fine.
So could it be a bug with @Value ?

Edit: The problem may not be with @Value but with SpEL because the following is not working neither :
    @SqsListener(value = {
        "${x.y.z}"
    })

Edit: It may be related to a missing "/" at the end of the config import, as mentioned in https://github.com/awspring/spring-cloud-aws/issues/696, I am going to give it a try.

Edit: I confirm it solved the problem. So, a couple of comments, first it's strange that it's working fine for Spring properties but not for custom properties, second there should be a clear comment about that point in the documentation. Thanks !

**Comment by maciejwalkowiak** [comment]
@Cimballi you're welcome to submit a PR to update docs
